src/Control/Carrier/Empty/Maybe.hs

```diff
@@ -27,7 +27,7 @@ runEmpty = runMaybeT . runEmptyC
 {-# INLINE runEmpty #-}
 
 newtype EmptyC m a = EmptyC { runEmptyC :: MaybeT m a }
-  deriving (Applicative, Functor, Monad, MonadFix, MonadIO, MonadTrans)
+  deriving (Applicative, Functor, Monad, MonadFix, MonadIO, MonadTrans, Show)
```

Comment:
We now generate these in the tests, and being able to show them is handy (vs. using `Blind` all over the place).